lib/cosmos/packet_logs/packet_log_writer.rb

```diff
@@ -66,7 +73,10 @@ def initialize(
       @label = label
       @logging_enabled = ConfigParser.handle_true_false(logging_enabled)
       @cycle_time = ConfigParser.handle_nil(cycle_time)
-      @cycle_time = Integer(@cycle_time) if @cycle_time
+      if @cycle_time
+        @cycle_time = Integer(@cycle_time)
+        raise "cycle_time must be >= #{CYCLE_TIME_INTERVAL}" if @cycle_time < CYCLE_TIME_INTERVAL
```

Comment:
Calling Integer() on a float less than 1 results in 0 which was a potential bug. Also it simply needs to be >= CYCLE_TIME_INTERVAL